app/components/document_title_component.html.erb

```diff
@@ -5,7 +5,7 @@
     <%= title -%>
   <% end %>
 
-  <div class="object-type object-type-<%= object_type %> fw-bold text-center text-uppercase rounded-pill fs-4 mt-2 float-end">
-    <%= object_type %>
+  <div class="object-type object-type-<%= object_type_class %> fw-bold text-center text-uppercase rounded-pill fs-4 mt-2 float-end">
```

Comment:
Now that we have an object type with a space in it (`virtual object`), we need a wee routine for deriving the class from the label.